tango/common/registrable.py

```diff
@@ -184,7 +184,12 @@ def search_modules(cls: Type[_RegistrableT], name: str):
         def try_import(module):
             try:
                 import_module_and_submodules(module)
-            except (ModuleNotFoundError, ImportError):
+            except ModuleNotFoundError as e:
+                if e.name == module:
+                    pass
+                else:
+                    raise
+            except ImportError:
```

Comment:
This is causing some test failures

Reply:
Did we rely on `ImportError` being silently swallowed to make tests work?

Reply:
We need this because we might try to import an integration that isn't installed, i.e. an integration that is missing some dependencies, like torch.